### PR TITLE
[16.0][MIG] stock_picking_import_serial_number: Migration to 16.0

### PR DESCRIPTION
**Pull request by victoralmau**
Migration to 16.0

Please @sergio-teruel and @carlosdauden can you review it?

@Tecnativa TT47693

### CODE REVIEW

File: stock_picking_import_serial_number/tests/common.py [hunk 1]
```diff
@@ -17,8 +17,7 @@ def assertUniqueIn(self, element_list):
     @classmethod
     def setUpClass(cls):
         super().setUpClass()
-        cls.env = cls.env(context=dict(cls.env.context, tracking_disable=True))
-        cls.lot_obj = cls.env["stock.production.lot"]
+        cls.lot_obj = cls.env["stock.lot"]
```
Comment (rousseldenis) [inline stock_picking_import_serial_number/tests/common.py:20]: Here above you can simplify writing by:

`class CommonStockPickingImportSerial:`

**Review by rousseldenis — APPROVED** [review]
Code review. Minor comment

**Review by carlosdauden — APPROVED** [review]

**Pull request merged**